bin/rucio-admin

```diff
@@ -489,6 +489,25 @@ def list_rses(args):
     return SUCCESS
 
 
+@exception_handler
+def update_rse(args):
+    """
+    %(prog)s update [options] <field1=value1 field2=value2 ...>
+
+    Update the settings of the RSE ('availability_read', 'availability_write', 'availability_delete', 'latitude', 'longitude', 'time_zone', 'rse_type',
+    'volatile', 'deterministic', 'region_code', 'country_name', 'city', or 'staging_area')
```

Comment:
Maybe remove `volatile`, `deterministic`, `rse_type` and `staging_area` from this list (also below). This is not necessarily something which should be changed post-creation. (Not sure if the core actually blocks this)